content/docs/deployment/csminstallationwizard/src/templates/helm/csm-1.12.0-values.template

```diff
@@ -382,7 +382,7 @@ csi-isilon:
     authorization: 
       image: dellemc/csm-authorization-sidecar:v1.12.0
     metadataretriever: 
-      image: dellemc/csi-metadata-retriever:v1.8.0
+      image: dellemc/csi-metadata-retriever:v1.9.0
```

Comment:
Path changes in installation wizard are taken care in this [PR](https://github.com/dell/csm-docs/pull/1296/files#diff-8c916df10fa4b13e493875bfac6c8ad7658fb3040242b671f34c357c2edf73ceR385)